node/src/BaseClient.ts

```diff
@@ -965,9 +965,11 @@ export class BaseClient {
      * @remarks Since Valkey version 6.2.0.
      *
      * @param key - The key to retrieve from the database.
-     * @param options - (Optional) Set expiriation to the given key.
-     *                  "persist" will retain the time to live associated with the key. Equivalent to `PERSIST` in the VALKEY API.
-     *                  Otherwise, a {@link TimeUnit} and duration of the expire time should be specified.
+     * @param options - (Optional) Additional Parameters:
+     * - Set expiriation to the given key:
+     * "persist" will retain the time to live associated with the key. Equivalent to `PERSIST` in the VALKEY API.
+     * Otherwise, a {@link TimeUnit} and duration of the expire time should be specified.
+     * - {@link Decoder} type which defines how to handle the response. If not set, the default decoder from the client config will be used.
```

Comment:
We have another doc for decoder, copy from the wiki page https://github.com/valkey-io/valkey-glide/wiki/Node:-Add-support-to-encoded-values

Reply:
Update everywhere